ols/src/cache/redis_cache.py

```diff
@@ -74,10 +78,12 @@ def insert_or_append(self, user_id: str, conversation_id: str, value: str) -> No
                 memory is higher than maxmemory.
         """
         key = super().construct_key(user_id, conversation_id)
-
         old_value = self.get(user_id, conversation_id)
         with self._lock:
             if old_value:
-                self.redis_client.set(key, old_value + "\n" + value)
+                old_value.append(value)
```

Comment:
is this list guaranteed to stay ordered through redis storage + retrieval?

Reply:
yes